client/my-sites/checkout/checkout-thank-you/atomic-store-thank-you-card.jsx

```diff
@@ -50,24 +50,21 @@ class AtomicStoreThankYouCard extends Component {
 
 		this.setState( { resendStatus: RESEND_PENDING } );
 
-		wpcom
-			.undocumented()
-			.me()
-			.sendVerificationEmail( ( error ) => {
-				if ( error ) {
-					this.props.errorNotice(
-						translate( "Couldn't resend verification email. Please try again." ),
-						{
-							id: VERIFY_EMAIL_ERROR_NOTICE,
-						}
-					);
-
-					this.setState( { resendStatus: RESEND_ERROR } );
-					return;
-				}
-
-				this.setState( { resendStatus: RESEND_SUCCESS } );
-			} );
+		wpcom.req.post( '/me/send-verification-email', ( error ) => {
```

Comment:
This would be easy to convert to `.then().catch()`

Reply:
Yeah. I'm just going to keep it as-is to keep the commit and make the rebase of the dependent PR easier, if you don't mind.